2-vault-aws-dynamic-secrets-demo.sh

```diff
@@ -75,5 +75,6 @@ pe "vault lease revoke aws/creds/my-role/$(tail -1 leases.txt)"
 pei ""
 pe "vault secrets disable aws"
 pe ""
+pe "aws iam list-users"
```

Comment:
Show that users created by Vault are automatically removed when the leases are revoked, which happens for all leases when engine is disabled.